rp2040-hal/src/i2c.rs

```diff
@@ -80,6 +83,38 @@ pub enum Error {
     AddressReserved(u16),
 }
 
+#[cfg(feature = "eh1_0_alpha")]
+impl core::fmt::Debug for Error {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
```

Comment:
Use the same name parameter name `fmt` for the formatter for both core::fmt::Formatter and defmt, no reason these should be different

Reply:
The difference comes from how the parameter is called in the respective trait definitions. 
Any reason to chose `fmt` over `f` ?

Reply:
No reason, just would prefer that it's the same one for consistency. 
I don't believe in making your project conform to the internal naming conventions of libraries.
Use f for both if you prefer brevity, fmt for both would be fine, as would formatter.

Reply:
opted for `fmt` as requested :)